codemod/Types.d.ts

```diff
@@ -0,0 +1,8 @@
+export {};
+
+declare global {
+  interface Blob {
```

Comment:
We're going to have problems with this because the typescript linter's not going to like it.

You are extending the interface only for this project which means that if I import your library into another project the `Blob` interface will not be extended with the types you added.

Reply:
Removed in favor of the other solution you provided.